File: android/app/src/main/java/com/nexusbuild/apps/fliprate/util/PopularPairs.java
package com.nexusbuild.apps.fliprate.util;

import com.nexusbuild.apps.fliprate.model.FavoritePair;

import java.util.Arrays;
import java.util.List;

public class PopularPairs {
    public static final List<FavoritePair> LIST = Arrays.asList(
            new FavoritePair("USD", "EUR"),
            new FavoritePair("USD", "GBP"),
            new FavoritePair("USD", "BDT"),
            new FavoritePair("EUR", "BDT"),
            new FavoritePair("GBP", "BDT"),
            new FavoritePair("USD", "JPY"),
            new FavoritePair("USD", "INR")
    );
}

File: web/js/popularpairs.js
const POPULAR_PAIRS = [
  { from: "USD", to: "EUR" },
  { from: "USD", to: "GBP" },
  { from: "USD", to: "BDT" },
  { from: "EUR", to: "BDT" },
  { from: "GBP", to: "BDT" },
  { from: "USD", to: "JPY" },
  { from: "USD", to: "INR" },
];

function renderPopularPairs() {
  const listEl = document.getElementById("popular-pairs-list");

  POPULAR_PAIRS.forEach((pair) => {
    const li = document.createElement("li");

    const label = document.createElement("button");
    label.type = "button";
    label.className = "pair-label";
    label.textContent = `${pair.from} → ${pair.to}`;
    label.setAttribute("aria-label", `Load ${pair.from} to ${pair.to}`);
    label.addEventListener("click", () => {
      setConverterPair(pair.from, pair.to);
    });

    li.appendChild(label);
    listEl.appendChild(li);
  });
}

function initPopularPairs() {
  renderPopularPairs();
}
